var/spack/repos/builtin/packages/snappy/package.py

```diff
@@ -47,6 +48,11 @@ def cmake_args(self):
 
         return args
 
+    def setup_environment(self, spack_env, run_env):
+        if '+pic' in self.spec:
+            spack_env.append_flags('CFLAGS', self.compiler.pic_flag)
+            spack_env.append_flags('CXXFLAGS', self.compiler.pic_flag)
```

Comment:
See https://github.com/spack/spack/pull/8543#issuecomment-399606529 for a more robust way to handle this. This is specific to `CMakePackage`.

Reply:
Thanks, changed!